homeassistant/components/habitica/strings.json

```diff
@@ -57,13 +57,51 @@
           "rogue": "Rogue"
         }
       }
+    },
+    "todo": {
+      "todos": {
+        "name": "To-Do's"
+      },
+      "dailys": {
```

Comment:
I dislike the typo in here, but it probably originates from their service

Reply:
yapp, thats how they write it